app/helpers/application_helper.rb

```diff
@@ -1,2 +0,0 @@
-module ApplicationHelper
-end
```

Comment:
**Query**: Why is this file deleted? IMHO, we should keep this file.

Reply:
i thought it isn't necessary. I recreated it, but why is this helper file needed and all others should be deleted?

Reply:
We are deleting all other helper files because right now we do not need any model/controller specific helpers. But, ApplicationHelper is a common/base class, like ApplicationController. It is a convention that we follow. Thanks!